content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -0,0 +1,98 @@
+
+To collect logs from applications running in GCE or GKE, you can also use the [Datadog Agent][13].
+
+## Setup
+
+### Prerequisites
+
+The [Google Cloud Platform integration][1] is successfully installed.
+
+### Create a Cloud Pub/Sub topic
+
+1. Go to the [Cloud Pub/Sub console][2] and create a new topic.
+
+    {{< img src="integrations/google_cloud_platform/create_a_topic.png" alt="Create a topic" style="width:80%;">}}
+
+2. Give that topic an explicit name such as `export-logs-to-datadog` and click _Create_.
+
+**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][3]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [the Monitor the Log Forwarding section](#monitor-the-log-forwarding) for information on setting up monitor notifications if you approach those limits.
+
+### Subscribe to your Google Cloud Pub/Sub topic
+
+1. In the [Cloud Pub/Sub console][2], select `Subscriptions` in the left hand navigation. Click `Create Subscription`.
+2. Create a subscription ID and select the topic you previously created.
+3. Select the `Push` method and enter the following command, replacing `<DATADOG_API_KEY>` with the value of a valid [Datadog API key][4]: 
+```
+https://gcp-intake.logs.{{< region-param key="dd_site" code="true" >}}/api/v2/logs?dd-api-key=<DATADOG_API_KEY>&dd-protocol=gcp
+```
+4. Configure any additional options, such as **Subscription expiration**, **Acknowledgment deadline**, **Message retention duration**, or **Dead lettering**.
+5. Under **Retry policy**, select `Retry after exponential backoff delay`.
+6. Click `Create` at the bottom.
+
+The Pub/Sub is ready to receive logs from Google Cloud Logging and forward them to Datadog.
+
+### Export logs from Google Cloud
+
+1. Go to [the Google Cloud Logs Explorer page][6] and filter the logs that need to be exported.
+2. From the **Log Router** tab, select **Create Sink**.
+3. Provide a name for the sink.
+4. Choose _Cloud Pub/Sub_ as the destination and select the pub/sub that was created for that purpose.
+   **Note**: The pub/sub can be located in a different project.
+
+    {{< img src="integrations/google_cloud_pubsub/creating_sink2.png" alt="Export Google Cloud Pub/Sub Logs to Pub Sub" >}}
+
+5. Click **Create Sink** and wait for the confirmation message to appear.
+
+**Note**: It is possible to create several exports from Google Cloud Logging to the same Pub/Sub with different sinks.
+
+**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][3]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [the Monitor the Log Forwarding section](#monitor-the-log-forwarding) for information on setting up monitor notifications if you approach those limits.
```

Comment:
Do we really want to duplicate this warning here? Feels like the first one should be enough?

Reply:
Thanks for flagging that, I agree it's excessive and removed it in [4b298a1](https://github.com/DataDog/documentation/pull/19467/commits/4b298a1fb4a1eee9caf5adc9acc42dec4cf798fe)